wear/src/main/java/com/yannik/anki/PullButton.java

```diff
@@ -242,7 +242,7 @@ public boolean onTouch(final View v, MotionEvent event) {
                     mVelocityTracker.computeCurrentVelocity(1);
                     float yVelocity = mVelocityTracker.getYVelocity();
 
-                    if (viewPositionY < minMovementDistance && yVelocity >= 0) {
+                    if (viewPositionY < minMovementDistance && ((upsideDown && yVelocity >= 0) || (!upsideDown && yVelocity <= 0))) {
```

Comment:
Sorry for the somewhat messy line, I didn't want to mess with the rest of the if-statement structure.